src/components/graph/graph.config.js

```diff
@@ -97,6 +97,9 @@
  *
  * **[note]** react-d3-graph will map this values to [d3 symbols](https://github.com/d3/d3-shape#symbols)
  * <br/>
+ * @param {number} [node.gravity=-100] - this will define how close nodes are to each other.
```

Comment:
I would like to see this values isolated in the scope of a `d3` object property, can you please move this to:

```javascript
...
d3: {
    gravity: -100,
    linkSize: 100
},
...
```